cernopendata/views.py

```diff
@@ -54,6 +65,48 @@ def search_wrapper():
         return invenio_search_view()
 
 
+def translate_search_url(args, facets):
+    """Translate old search querystring args to new ones."""
+    subagg_agg_mapping = {}
```

Comment:
What do you think of breaking down this function into smaller pieces? It got pretty heavy and it's quite hard to follow all the logic, maybe some comments would be beneficial as well. In case someone has to debug it in the future